### DISCUSSION TITLE
Presidio Text Size Limit

### DISCUSSION THREAD

**Discussion by abdullahalrifat** [microsoft/presidio#916] [Q&A]
what is the in general text size that gives optimum result as i can have big texts so thinking of breaking them and pushing to presidio to analyze. 
i also looked presidio gives exception with big texts and also container dies often so what is the text limit here?

**Comment by omri374** [comment]
Hi @abdullahalrifat,
Presidio itself does not have an explicit text length limit. The underlying models used might. 
Could you please share more information on the issue you are facing?

**Reply by abdullahalrifat** [reply]
i am having continious text(10 message in a second) and (some time length 1M +), we are slicing the text into 1000 char limit chunks and passing it to presidio en_core_web_lg model but often presidio just dies and we get exception on our endpoint saying request time out. for en_core_web_trf this model, exception rate is very high. i think it can be related to the no of messages we are sending to presidio it can be overwhelming or the text length combinely it's unabel to process this much request with that long text. do you suggest any number that can be handled?
